content/docs-community/concepts/contributing-to-docs/terms/reviews/reviews.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Reviews'
+Description: 'Reviews are conducted to ensure valid content is merged into the Docs repository.'
+Subjects:
+  - 'Code Foundations'
+Tags:
+  - 'Documentation'
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'introduction-to-open-source'
+  - 'paths/code-foundations'
+---
+
+**Reviews** are conducted by members of the Curriculum Free Team (CFT) at Codecademy who help maintain the [`docs` repository](https://www.codecademy.com/resources/docs/docs-community/docs-repository).
+
+## The Review Process
+
+When a new pull request (PR) is opened by a contributor, a CFT member will flag it for review by doing the following:
+
+- Add any relevant labels to the PR.
+- Link any existing, relevant issues.
+- Request/assign other CFT members for review.
+
+A CFT member may view the PR files, make comments, and suggest changes.
+
+### Number of Reviews
+
+Most commonly, PRs undergo 1-3 reviews to ensure each entry gets an ample amount of feedback and opportunity for improvement. The number of reviews for a given PR is at the discretion of CFT members considering factors such as the following:
+
+- The content of the PR file(s) warrants further review by a separate, subject matter expert.
+- There is a considerable amount of edits to existing Docs content.
+- The PR includes content for a new Docs topic (in addition to concepts and terms).
+
+### Invalid & Duplicate PRs
+
+There are some cases where a PR will be closed by a CFT member due to one or more factors.
+
+A PR is closed and marked as a "duplicate" if it pertains to the same file(s) as another existing PR.
+
+A PR is closed and marked as "invalid" for one or more of the following:
```

Comment:
I wonder if we should adopt different verbiage here in light of the hacktober issue. Perhaps we should us 'void' or 'inadmissable'

Reply:
We also have a "won't fix" label, fwiw.